### ISSUE TITLE
DLNA - Device disappeared when trying to call cmd_play since v2.4b4 & v2.4b6

### ISSUE THREAD

**Issue by bbs2web** [music-assistant/hass-music-assistant#3210]
### What version of Music Assistant has the issue?

2.4b6

### What version of the Home Assistant Integration have you got installed?

2024.11.4

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

Have 3 x Up2Stream 2.0 (v4.0) DLNA players which no longer stream or occasionally stutter burps of audio (as in silent with very brief cuts of audio (estimate 100ms or less) being played). Can continuously ping the player from Home Assistant.

Worked perfectly on version 2.3.0b34

Sample log entry:
2024-11-21 21:31:52.147 WARNING (MainThread) [music_assistant.dlna] Device disappeared when trying to call cmd_play

I've tried playing YouTube Music as well as Internet Radio (Cafe Del Mar).

### How to reproduce

Play either YouTube Music or Internet Radio and stream via DLNA. Either no audio via either or some YouTube music has very occasional tiny snippets of audio (estimate <100ms).

### Music Providers

![image](https://github.com/user-attachments/assets/e2ec90b8-3cd0-4f26-9058-192140a9da47)

### Player Providers

![image](https://github.com/user-attachments/assets/fae95924-2774-45de-8629-be8a0c4f2c66)

### Full log output

[2024-11-21 211742.994 INFO (MainThr.txt](https://github.com/user-attachments/files/17851305/2024-11-21.211742.994.INFO.MainThr.txt)

### What version of Home Assistant Core are your running

2024.11.2

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by bbs2web** [comment]
Just upgraded to Music Assistant 2024.11.5 and the problem unfortunately remains.

**Comment by OzGav** [comment]
Per the troubleshooting docs have you tried the suggestions on the DLNA page? If so what happened?

**Comment by bbs2web** [comment]
I had tried lossy compression (MP3) playback and switched between the 3 streaming formats. Looking in to this further shows the device only having a signal strength of -90 dBm and only syncing between 1 and 2 Mbps.

It appears that this device's WiFi chip somehow failed on the same night that I had upgraded Music Assistant from 2.3 to 2.4. Use it daily and it had worked earlier that afternoon, prior to me running updates before turning in for the night...

Thanks for an amazing project, really love the constant improvements!
